icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/ULocaleTest.java

```diff
@@ -5154,4 +5154,138 @@ public void Test20900() {
             Assert.assertEquals(testData[row][1], loc.toLanguageTag());
         }
     }
+    @Test
+    public void TestCanonical() {
+        // Test replacement of languageAlias
+
+        // language _ variant -> language
+        Assert.assertEquals("nb",
+            ULocale.createCanonical(
```

Comment:
please make & use a static private helper function (just before or after this test function) for

ULocale.createCanonical(ULocale.forLanguageTag(languageTag)).toLanguageTag())

use a short name for the helper function to keep the test cases readable; maybe "canonicalTag()"?

Reply:
done